src/electionguard/guardian.py

```diff
@@ -95,20 +95,12 @@ class PrivateGuardianRecord:
     election_keys: ElectionKeyPair
     """Private election Key pair of this guardian"""
 
-    backups_to_share: Dict[GuardianId, ElectionPartialKeyBackup]
```

Comment:
See this issue. The private record that is produced should have just the raw values and the not the entire backup. 

This will also change how the constructor works and what information is saved. 

https://github.com/microsoft/electionguard-python/issues/607

Reply:
The private record builder looks solid but the nonce should still be isolated into its own class method `from_nonce`. The nonce + election key is a redundant constructor.

Reply:
Please clarify: I don't understand what action the first comment requires, or if that's just background information.  

As far as the second comment about Nonce I took a stab at it here: https://github.com/microsoft/electionguard-python/pull/608/commits/f8cb5d0b82a44b04b0501274460aaaf92b3c0edb.  I don't love that `generate_key_pair` parameter.  I can remove it if you don't mind that `generate_election_key_pair` would get called twice when the constructor is called from the `from_nonce` static method.  

Incidentally, I would normally solve this kind of thing by having two constructors, one with nonce, one with keys, but I couldn't figure out how to do that.  Not sure what I'm missing, but would welcome guidance if this approach isn't what you're looking for.

Reply:
Per our conversation I:
- Switched from staticmethods to classmethods for constructors
- Made a from_nonce overload
- Restored all removed fields from PrivateGuardianRecord
- Moved `generate_election_key_pair` out of the constructor

To accomplish that last point, I had to make a new `@classmethod` type constructor which I called `from_context_info` for lack of a better name that takes neither nonce, nor key data, but then calls `generate_election_key_pair`.  I then refactored all code in the codebase that creates a guardian using the old constructor and switched it to `Guardian.from_context_info(...)`.